VERSION "1.0"

NS_ :
    NS_DESC_
    CM_
    BA_DEF_
    BA_
    VAL_
    CAT_DEF_
    CAT_
    FILTER
    BA_DEF_DEF_
    EV_DATA_
    ENVVAR_DATA_
    SGTYPE_
    SGTYPE_VAL_
    BA_DEF_SGTYPE_
    BA_SGTYPE_
    SIG_TYPE_REF_
    VAL_TABLE_
    SIG_GROUP_
    SIG_VALTYPE_
    SIGTYPE_VALTYPE_
    BO_TX_BU_
    BA_DEF_REL_
    BA_REL_
    BA_DEF_DEF_REL_
    BU_SG_REL_
    BU_EV_REL_
    BU_BO_REL_
    SG_MUL_VAL_

BS_:

BU_: MASTER_NODE HEAT_PLATE_NODE_1 MOTOR_NODE_1 TEMP_NODE_1 TEMP_NODE_2

BO_ 2147549088 HEAT_PLATE_CMD_1: 8 MASTER_NODE
 SG_ RELAY_STATE : 0|1@1+ (1,0) [0|1] "" HEAT_PLATE_NODE_1

BO_ 2147549089 HEAT_PLATE_STATE_1: 8 HEAT_PLATE_NODE_1
 SG_ RELAY_STATE : 0|1@1+ (1,0) [0|1] "" MASTER_NODE

BO_ 2147549104 MOTOR_CMD_1: 8 MASTER_NODE
  SG_ RELAY_STATE : 0|1@1+ (1,0) [0|1] "" MOTOR_NODE_1

BO_ 2147549105 MOTOR_STATE_1: 8 MOTOR_NODE_1
  SG_ RELAY_STATE : 0|1@1+ (1,0) [0|1] "" MASTER_NODE

BO_ 2147549121 TEMP_STATE_1: 8 TEMP_NODE_1
  SG_ TEMP_C : 0|16@1- (0.01,0) [-327.68|327.67] "C" MASTER_NODE
  SG_ TEMP_V : 16|16@1+ (0.001,0) [0|65.535] "V" MASTER_NODE

 BO_ 2147549122 TEMP_STATE_2: 8 TEMP_NODE_2
   SG_ TEMP_C : 0|16@1- (0.01,0) [-327.68|327.67] "C" MASTER_NODE
   SG_ TEMP_V : 16|16@1+ (0.001,0) [0|65.535] "V" MASTER_NODE
